examples/deployments/aws-terraform/variables.tf

```diff
@@ -86,3 +107,39 @@ variable "chroma_data_volume_size" {
   type        = number
   default     = 20
 }
+
+variable "prevent_chroma_data_volume_delete" {
```

Comment:
Let's remove this variable since it's not used anywhere. Then this LGTM

Reply:
Done